.github/workflows/mddiffcheck.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Check CAP Diffs'
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  mddiffcheck:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1
```

Comment:
If you need it, I _think_ that you may want `go-version: '^1.'` ? I looked at https://github.com/actions/setup-go and I am not sure what kind of matching is happening here

Reply:
This uses the https://www.npmjs.com/package/semver package and will resolve 1 to the latest v1 that is installed, which is 1.16. But you're right, we should be using the caret to specify 1.16 or newer, because the `mddiffcheck` tool uses functions that were added to the stdlib of 1.16. Updated in e46bce9.